### PR TITLE
mahage-ediscovery-async-holds 

### PR DESCRIPTION
**Pull request by mahage-msft**
.md PR - https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/9984
code PR - https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5573519

### CODE REVIEW

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "custodian: applyHold"
+description: "Apply a legal hold to a custodian"
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-applyhold.md:3]: Please, add a period at the end of this sentence.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Request headers
+
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+
+## Request body
+
+In the request body, supply JSON representation of the parameters.
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-applyhold.md:49]: Please, add "a" before "JSON".

Please, apply this change also here:

- api-reference/beta/api/ediscovery-custodian-removehold.md
- api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|ids|String collection|List of custodian Ids for whom to apply holds to.|
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-applyhold.md:55]: In regular text, please use ID as the abbreviation for identity. => ...custodian IDs for...

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Examples
+
+### Request
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-applyhold.md:64]: For consistency, please add the following boilerplate text here:

```
### Request

The following is a an example of a request.

<!-- {
```


Please, apply this change also here:

- api-reference/beta/api/ediscovery-custodian-removehold.md
- api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+{
+  "ids": [
+    "2192ca408ea2410eba3bec8ae873be6b",
+    "ab9ec130f90248949f09a1a7032f5352"
+  ]
+}
+```
+
+### Response
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-applyhold.md:85]: For consistency, please add the following boilerplate text here:

```
### Response

The following is a an example of the response.

<!-- {
```

Please, apply this change also here:

- api-reference/beta/api/ediscovery-custodian-removehold.md
- api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md

File: api-reference/beta/api/ediscovery-custodian-removehold.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
+title: "custodian: removeHold"
+description: "Removes all holds from the specified custodians"
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-removehold.md:3]: 1) The description for an API topic should start with an imperative verb. In this case, please replace "Removes" with "Remove". 

2) Also, please add a period at the end of this description.


=> Please, apply these changes also to line 16. <=

File: api-reference/beta/api/ediscovery-custodian-removehold.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+## Request body
+
+In the request body, supply JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|ids|String collection|List of custodian Ids for whom to remove holds from.|
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-custodian-removehold.md:55]: In regular text, please use ID as the abbreviation for identity. => ...custodian IDs for...

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
+---
+title: "noncustodialDataSource: applyHold"
+description: "Apply a legal hold to a noncustodialDataSource"
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:3]: 1. For consistency, please add "list of" after "to a" => ...legal hold to a list of...

2. Please, add a period at the end of this sentence.

3. Please, add "s" after "noncustodialDataSource" => noncustodialDataSources

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# noncustodialDataSource: applyHold
+
+Namespace: microsoft.graph.ediscovery
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Apply a legal hold to a list of [noncustodialDataSource](../resources/ediscovery-noncustodialdatasource.md)s.
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:16]: For consistency, please add that "s" after "noncustodialDataSource"  => \[noncustodialDataSources\](../resources/ediscovery-noncustodialdatasource.md).

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+## Request body
+
+In the request body, supply JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|ids|String collection|List of non-custodial data source ids Ids for whom to apply holds to.|
```
Comment (JarbasHorst) [inline api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:55]: Please, replace "ids Ids" with "IDs".

File: api-reference/beta/api/ediscovery-custodian-applyhold.md (hunk 1 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:3]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md (hunk 2 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:49]: Fixed in 3 places.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md (hunk 10 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:55]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md (hunk 4 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:64]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md (hunk 5 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:85]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-removehold.md (hunk 7 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-removehold.md:55]: Fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md (hunk 8 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:3]: Fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md (hunk 9 above)
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:16]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md (hunk 3 above)
Comment (JarbasHorst) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md]: Yes, that is correct. The feedback was only about the description. Parameter and property names should be in camel case format.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 11]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the **holdStatus** property to [dataSourceContainer](https://docs.microsoft.com/en-us/graph/api/resources/dataSourceContainer?view=graph-rest-beta) resource.",
+					"Target": "dataSourceContainer"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "applyHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the custodian resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:662]: Please, add a reference link to custodian.

Same for line 694.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 12]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the custodians resource collection.",
+					"Target": "applyHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "applyHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the non custodial data source resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:678]: 1) Currently, your PR doesn't include this change. Please add the "applyHold" method to the "Methods" table of the noncustodialDataSource resource topic.

2) Please, replace "non custodial data source" with "noncustodialDataSource" and add a reference link to it.

   Same for line 710.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 13]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [caseHoldOperation](https://docs.microsoft.com/en-us/graph/api/resources/caseHoldOperation?view=graph-rest-beta) resource",
+					"Target": "caseHoldOperation"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Property",
+					"ChangedApiName": "holdStatus",
+					"ChangeType": "Addition",
+					"Description": "Added the **holdStatus** property to [dataSource](https://docs.microsoft.com/en-us/graph/api/resources/dataSource?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:646]: 1) Your PR doesn't include this change. However, the **holdStatus** property was added to the custodian resource. In this case, please update this description.

2) After applying the change in 1, please add "the" before "[custodian]" =>  ...property to the [custodian]...

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 14]
```diff
@@ -620,6 +620,112 @@
...
+					"Target": "dataSource"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Property",
+					"ChangedApiName": "holdStatus",
+					"ChangeType": "Addition",
+					"Description": "Added the **holdStatus** property to [dataSourceContainer](https://docs.microsoft.com/en-us/graph/api/resources/dataSourceContainer?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:654]: Your PR doesn't include this change. In this case, this entry should be removed from the changelog.

File: api-reference/beta/resources/ediscovery-custodian.md [hunk 15]
```diff
@@ -22,7 +22,6 @@ In the context of eDiscovery, represents a user and all of their digital assets,
 |[List custodians](../api/ediscovery-case-list-custodians.md)|[microsoft.graph.ediscovery.custodian](../resources/ediscovery-custodian.md) collection|Get a list of **custodian** objects and their properties.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/ediscovery-custodian.md:22]: Please add the "applyHold" and "removeHold" methods to this "Methods" table.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 16]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the custodian resource.",
+					"Target": "applyHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "applyHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the custodians resource collection.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:670]: Your PR doesn't include this change. In this case, this entry should be removed from the changelog.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 17]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the custodian resource collection.",
+					"Target": "removeHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "removeHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the non custodial data source resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:710]: 1) Currently, your PR doesn't include this change. Please add the "removeHold" method to the "Methods" table of the noncustodialDataSource resource topic.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 18]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the non custodial data source resource.",
+					"Target": "applyHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "applyHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [applyHold](https://docs.microsoft.com/en-us/graph/api/applyHold?view=graph-rest-beta) method to the non custodial data source resource collection.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:686]: Your PR doesn't include this change. In this case, this entry should be removed from the changelog.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 19]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the custodian resource.",
+					"Target": "removeHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "removeHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the custodian resource collection.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:702]: Your PR doesn't include this change. In this case, this entry should be removed from the changelog.

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 20]
```diff
@@ -620,6 +620,112 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the non custodial data source resource.",
+					"Target": "removeHold"
+				},
+				{
+					"Id": "bd4ce6c3-840b-4f4d-a728-93223ef3bdb2",
+					"ApiChange": "Method",
+					"ChangedApiName": "removeHold",
+					"ChangeType": "Addition",
+					"Description": "Added the [removeHold](https://docs.microsoft.com/en-us/graph/api/removeHold?view=graph-rest-beta) method to the non custodial data source resource collection.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Compliance.Ediscovery.json:718]: Your PR doesn't include this change. In this case, this entry should be removed from the changelog.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Examples
+
+### Request
+
+The following is a an example of a request.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:65]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-applyhold.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+
+{
+  "ids": [
+    "2192ca408ea2410eba3bec8ae873be6b",
+    "ab9ec130f90248949f09a1a7032f5352"
+  ]
+}
+```
+
+The following is a an example of the response.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-applyhold.md:86]: Fixed - did that come from the doc stubs, HAHA!

File: api-reference/beta/api/ediscovery-custodian-removehold.md [hunk 23]
```diff
@@ -0,0 +1,99 @@
+---
+title: "custodian: removeHold"
+description: "Removes all holds from the specified custodians."
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-removehold.md:3]: Fixed

File: api-reference/beta/api/ediscovery-custodian-removehold.md [hunk 24]
```diff
@@ -0,0 +1,99 @@
...
+
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Examples
+
+### Request
+
+The following is a an example of a request.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-removehold.md:65]: Fixed.

File: api-reference/beta/api/ediscovery-custodian-removehold.md [hunk 25]
```diff
@@ -0,0 +1,99 @@
...
+
+{
+  "ids": [
+    "2192ca408ea2410eba3bec8ae873be6b",
+    "ab9ec130f90248949f09a1a7032f5352"
+  ]
+}
+```
+
+The following is a an example of the response.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-custodian-removehold.md:86]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+doc_type: apiPageType
+---
+
+# noncustodialDataSource: applyHold
+
+Namespace: microsoft.graph.ediscovery
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Apply a legal hold to a list of [noncustodialDataSources](../resources/ediscovery-noncustodialdatasource.md)s.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:16]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Examples
+
+### Request
+
+The following is a an example of a request.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:65]: Fixed

File: api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md [hunk 28]
```diff
@@ -0,0 +1,98 @@
...
+  "ids": [
+    "2192ca408ea2410eba3bec8ae873be6b",
+    "ab9ec130f90248949f09a1a7032f5352"
+  ]
+}
+```
+
+### Response
+
+The following is a an example of the response.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-applyhold.md:88]: Fixed

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 29]
```diff
@@ -0,0 +1,93 @@
...
+|ids|String collection|List of non-custodial data source ids Ids for whom to apply holds to.|
+
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Examples
+
+### Request
+
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:63]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 30]
```diff
@@ -0,0 +1,93 @@
...
+{
+  "ids": [
+    "2192ca408ea2410eba3bec8ae873be6b",
+    "ab9ec130f90248949f09a1a7032f5352"
+  ]
+}
+```
+
+### Response
+
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:84]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 31]
```diff
@@ -0,0 +1,93 @@
...
+## Request body
+
+In the request body, supply JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|ids|String collection|List of non-custodial data source ids Ids for whom to apply holds to.|
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:54]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+## Request headers
+
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+
+## Request body
+
+In the request body, supply JSON representation of the parameters.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:48]: fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+ms.prod: "ediscovery"
+doc_type: apiPageType
+---
+
+# noncustodialDataSource: removeHold
+Namespace: microsoft.graph.ediscovery
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Remove a legal hold to a list of [noncustodialDataSource](../resources/ediscovery-noncustodialdatasource.md)s.
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:15]: Fixed.

File: api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md [hunk 34]
```diff
@@ -0,0 +1,93 @@
+---
+title: "noncustodialDataSource: removeHold"
+description: "Remove a legal hold to a list of noncustodialDataSources"
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/api/ediscovery-noncustodialdatasource-removehold.md:3]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 35]
```diff
@@ -0,0 +1,69 @@
+---
+title: "caseHoldOperation resource type"
+description: "Represents the process of an eDiscovery case hold distribution."
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:3]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 36]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+# caseHoldOperation resource type
+
+Namespace: microsoft.graph.ediscovery
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the process of an eDiscovery case hold distribution. The **caseHoldOperation** can be retrieved from the `Location` header in the response when calling [custodian applyHolds](../api/ediscovery-custodian-applyhold.md), [custodian removeHolds](../api/ediscovery-custodian-removehold.md), [noncustodialDataSource applyHolds](../api/ediscovery-noncustodialdatasource-applyhold.md), or [noncustodialDataSource removeHolds](../api/ediscovery-noncustodialdatasource-removehold.md).
+
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:17]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 37]
```diff
@@ -0,0 +1,69 @@
...
+Inherits from [caseOperation](../resources/ediscovery-caseoperation.md).
+
+## Methods
+
+None
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:22]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 38]
```diff
@@ -0,0 +1,69 @@
...
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|action|[microsoft.graph.ediscovery.caseAction](../resources/ediscovery-caseoperation.md#caseaction-values)| The case action for this entity will always be `caseHold`. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
+|completedDateTime|DateTimeOffset| The date and time the export was completed.  Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:29]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 39]
```diff
@@ -0,0 +1,69 @@
...
+|createdBy|[identitySet](../resources/identityset.md)| The user who initiated the export operation. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
+|createdDateTime|DateTimeOffset| The date and time the export was created. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
+|id|String| The ID for the operation. Read-only. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:32]: fixed.

File: api-reference/beta/resources/ediscovery-caseholdoperation.md [hunk 40]
```diff
@@ -0,0 +1,69 @@
...
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.ediscovery.caseHoldOperation",
+  "baseType": "microsoft.graph.ediscovery.caseOperation",
+  "openType": false
+}
+-->
+
+``` json
+{
+  "@odata.type": "#microsoft.graph.ediscovery.caseHoldOperation",
```
[the remark this answers is not in the document]
Comment (mahage-msft) [reply api-reference/beta/resources/ediscovery-caseholdoperation.md:55]: Fixed. Can you ask the stub generator team to alphabatize this?

**Comment by Jarbas-MSFT** [comment]
According to Seungin Lyu, this PR can be closed.

**Pull request closed without merging**